docs/operations/security-user-auth.md

```diff
@@ -29,11 +29,12 @@ This document describes the Druid security model that extensions use to enable u
 
 At the center of the Druid user authentication and authorization model are _resources_ and _actions_. A resource is something that authenticated users are trying to access or modify. An action is something that users are trying to do. 
 
-There are three resource types:
+There are four resource types:
```

Comment:
heh this is actually a lie, there is also `VIEW`, but since there is no view implementation I don't think we need to call it out yet